### PR TITLE
update reqwest to 0.9

### PR DESCRIPTION
**Pull request by andy128k**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -14,26 +14,28 @@ version = "0.12.0"
...
-reqwest = { version = "0.8.0", optional = true }
+reqwest = { version = "0.9.0", optional = true }
+mime = { version = "0.3", optional = true }
 
 [dev-dependencies]
 version-sync = "0.5"
 
 [features]
-default = ["reqwest"]
+transport = ["reqwest", "mime"]
```
Comment (jonas-schievink) [inline Cargo.toml]: The name of this feature is a bit unfortunate - it seems to imply that the `Transport` trait can be disabled. Perhaps `http` would be a better name?

**Review by jonas-schievink — APPROVED** [review]
Thanks! The removal of the typed headers is pretty unfortunate, though, so unless you need it, I'd like to wait with the next crates.io release just to see if [they get added back in](https://github.com/hyperium/headers/pull/4).

**Comment by andy128k** [comment]
@jonas-schievink Currently the only issue with reqwest 0.8 is that OpenSSL 1.1.1 is not supported. 
I switched to my own implementation of `Transport`, so no hurry here.

**Comment by jonas-schievink** [comment]
Yeah I've been bitten by an outdated version of `openssl-sys` a few days ago, so I think it's still a good idea to release a new version with updated `reqwest`. The `reqwest` feature is still mentioned in the docs of the `http` module (and I think somewhere in `Request::call_url` - that one also has a `#[cfg]` that should probably point to `http` instead), can you update those docs to say `http` instead?

**Pull request merged**